src/components/Meta/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Head from 'next/head'
+
+interface Meta {
+  readonly title?: string
+  readonly keywords?: string
+  readonly description?: string
+}
+
+const Meta = ({ title, keywords, description }: Meta) => {
+  return (
+    <Head>
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <meta name="keywords" content={keywords} />
+      <meta name="description" content={description} />
+      <meta charSet="utf-8" />
+      <link rel="icon" href="/favicon.ico" />
+      <title>{title}</title>
+    </Head>
+  )
+}
+
+Meta.defaultProps = {
+  title: 'Cobda: Online Marketplace for Streetwear Products',
```

Comment:
I think only streetwear is meaningful enough. Anyway, I suggest these choices for your consideration. 

```suggestion
  title: 'Cobda: Online Marketplace for Buying and Selling Streetwear',
```
```suggestion
  title: 'Cobda: Online Streetwear Marketplace for everyone',
```

Reply:
Second one looks good for us